src/py/aspen/app.py

```diff
@@ -99,14 +102,47 @@ def logout():
 @application.route("/api/usergroup", methods=["GET"])
 @requires_auth
 def usergroup():
+
     with session_scope(application.config["DATABASE_CONFIG"].INTERFACE) as db_session:
-        # since authentication is required to access view this information is guaranteed to be in session
         profile = session["profile"]
-        user = (
-            db_session.query(User)
-            .options(joinedload(User.group))
-            .filter(User.auth0_user_id == profile["user_id"])
-            .one()
-        )
+        user = get_usergroup(db_session, profile["user_id"])
         user_groups = {"user": user.to_dict(), "group": user.group.to_dict()}
         return jsonify(user_groups)
+
+
+@application.route("/api/samples", methods=["GET"])
+@requires_auth
+def samples():
+
+    with session_scope(application.config["DATABASE_CONFIG"].INTERFACE) as db_session:
+        profile = session["profile"]
+        user = get_usergroup(db_session, profile["user_id"])
+
+        sequencing_reads = (
+            db_session.query(SequencingReads)
+            .join(Sample, Group, User)
+            .options(
+                joinedload(SequencingReads.sample).joinedload(Sample.submitting_group)
+            )
+            .filter(Sample.submitting_group == user.group)
+            .all()
+        )
+
+        all_sample_data = [
+            {"sequence": seq.to_dict(), "sample": seq.sample.to_dict()}
+            for seq in sequencing_reads
+        ]
+
+        # filter for only information we need in sample table view
+        filtered_sample_data = [
+            {
+                "private_identifier": s["sample"]["private_identifier"],
+                "public_identifier": s["sample"]["public_identifier"],
+                "upload_date": s["sequence"]["sequencing_date"],
```

Comment:
assuming the `sequencing_date` field will eventually be updated to say `upload_date` per your issue/story?

Reply:
@sidneymbell yes we'll update this to be upload date once we make that migration 👍, but for now i'm using as a proxy for `upload_date` since we currently don't have an `upload_date` field